specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json

```diff
@@ -0,0 +1,8511 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-09-01",
```

Comment:
Looks like it makes the same changes as in https://github.com/Azure/azure-rest-api-specs/pull/12350. Please apply the same comments left for the preview version.

Reply:
Changes are pushed.